cluster-autoscaler/metrics/metrics.go

```diff
@@ -50,6 +50,12 @@ const (
 	LogLongDurationThreshold = 5 * time.Second
 )
 
+const (
```

Comment:
Oh, and that actually passes the linter? Cool, I didn't know that, will start doing the same :)